### DISCUSSION TITLE
Rename "Services" to something else

### DISCUSSION THREAD

**Discussion by jimmykarily** [epinio/epinio#1280] [Ideas]
Our current services are more like "shared configuration" or "shared secrets" and less "services". Given we will work on real services soon (https://github.com/epinio/epinio/issues/1012) it may be a good idea to rename the old "services" to something else.

We should think if the output of the new "real" services will be a similar secret or something else.

**Comment by manno** [comment]
Are they used for something generic like mounting maps to pods or for sharing connection details between services?
What is their lifetime, is it coupled to services? Apparently not.

Maybe it's just "Configuration" that's mounted in the pods. 
API: `/configuration`, `epinio config create config1 abc 123`

**Reply by enrichman** [reply]
I like `Configuration`, it's clear and it doesn't conflicts with the k8s ConfigMap.

**Reply by jimmykarily** [reply]
`epinio config` already exists and it refers to Epinio's own configuration. Won't that be confusing?
